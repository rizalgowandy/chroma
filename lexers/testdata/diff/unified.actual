--- old.txt	2023-01-17 21:02:15.449417575 -0700
+++ new.txt	2023-01-17 21:02:12.489441682 -0700
@@ -1,5 +1,4 @@
-A
-A
 B
-C
+F
 D
+E
