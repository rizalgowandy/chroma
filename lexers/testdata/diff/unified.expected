[
  {"type":"GenericDeleted","value":"--- old.txt\t2023-01-17 21:02:15.449417575 -0700\n"},
  {"type":"GenericInserted","value":"+++ new.txt\t2023-01-17 21:02:12.489441682 -0700\n"},
  {"type":"GenericSubheading","value":"@@ -1,5 +1,4 @@\n"},
  {"type":"GenericDeleted","value":"-A\n-A\n"},
  {"type":"Text","value":" B\n"},
  {"type":"GenericDeleted","value":"-C\n"},
  {"type":"GenericInserted","value":"+F\n"},
  {"type":"Text","value":" D\n"},
  {"type":"GenericInserted","value":"+E\n"}
]
